### ISSUE TITLE
Dashboard is blank or not rendering

### ISSUE THREAD

**Issue by JerrettDavis** [Opticos/GWSL-Source#60]
On a fresh install with Windows 10 Pro `Build 21364.co_release.210416-1504`, opening the dashboard only renders the backdrop. It appears the buttons are present and clickable, but there's no text or images rendered on the backdrop. Even more interesting, when attempting to take a screenshot, the text IS visible!

![image](https://user-images.githubusercontent.com/2610199/115973158-8db24200-a518-11eb-9e1a-fc45918e9485.png)

**Comment by Pololot64** [comment]
I think you have hdr enabled. The way to go around this is to right click the gwsl icon, click Configure GWSL, and change acrylic_enabled to false instead of true. Then save. This is less pretty but I could not find another workaround

**Comment by JerrettDavis** [comment]
> I think you have hdr enabled. The way to go around this is to right click the gwsl icon, click Configure GWSL, and change acrylic_enabled to false instead of true. Then save. This is less pretty but I could not find another workaround

Looks like that solves the problem. Shame I lose out on the acrylic though. Thanks.

**Comment by Pololot64** [comment]
:( Maybe I can change it to a picture or color other than black in the next update

**Comment by JerrettDavis** [comment]
It's not a major issue, but maybe you can default to `acrylic` being off for new installations, with an `Enable Acrylic` option somewhere in the install wizard, or in the dashboard itself. That said, this is the first app, I've encountered rendering issues from with HDR enabled, I'm curious if it's coming form the underlying libraries, or if there's something more funky going on.

**Comment by Pololot64** [comment]
Part of it has to do with the fact that GWSL is written in python so I had to get real hacky about supporting acrylic. I don't want it to be disabled by default but maybe I can find a way to detect if hdr is on. I have not found it in the registry yet though

**Comment by JerrettDavis** [comment]
I'm sure it's not entirely ubiquitous, but it seems `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\VideoSettings\EnableHDRForPlayback` is a good starting point to detect HDR.

**Comment by Pololot64** [comment]
Maybe. I need to find an hdr display to test.. The other variables that exist for the same thing perplex me though. The registry is amazingly inconsistent between machines I discovered

**Comment by JerrettDavis** [comment]
Well, I happen to have several HDR displays to test with, so if you have any experimental builds, please send them my way, and I'd be happy to give them a go.

**Comment by Pololot64** [comment]
I can do that. Now I have to mess with the registry to find the real variable if it exists (because the one you mentioned is set to 1 on my pc and I don't have the problem)

**Comment by Pololot64** [comment]
And I can do a pretty good "fake acrylic" with the background image 
![image](https://user-images.githubusercontent.com/31389383/115975491-ce1abb80-a52a-11eb-86ef-584a98697a76.png)

**Comment by JerrettDavis** [comment]
I noticed you're down in Dallas, I'm just north of you in Tulsa, OK. If you'd send me an email or a DM me on twitter at @JDDavisPro, I'm happy to give you an HDR display to test with. 

As for the registry key, it's a bit misleading. Even if it's set to 1 on your local machine, it won't enable HDR unless windows detects an HDR compatible display. Disabling HDR on my displays (with the registry key set to 1) corrects the issue entirely, but obviously that's not optimal.

**Comment by Pololot64** [comment]
Yeah. And thanks that is very thoughtful but I know someone I can borrow one from.  ;)

**Comment by JerrettDavis** [comment]
Sounds good. Feel free to close the issue (or tag it for future reference). I'm able to use the app with acrylic disabled, and that's what matters most.

**Comment by Pololot64** [comment]
👍I will keep it open for a bit though so it can benefit others. I will follow up in a few days or so if I find a fix or have a better build

**Comment by ReidWeb** [comment]
This issue appears to be presenting itself in the shortcut creator too?

**Comment by wmsphd** [comment]
i've had this problem on one of my PCs since i first installed gwsl months/years ago and want to try again to find a solution.
as in the original post, the dashboard is blank, but shows up in screenshots. I have disabled hdr to no avail. if i set acrylic_enabled to false, or make ANY change in the config i get no dashboard at all, just a taskbar icon. i use bitdefender firewall, malwarebytes and have both start11 and explorerpatcher installed. i have tried disabling both security packages but it didn't help. Win11 23h2 all updates on an asus rog flow z13 used in portrait mode with no keyboard [i.e. as a tablet]. Any help will be appreciated!

**Comment by Pololot64** [comment]
I'm gonna try to release an update for this before the end of the summer.

Edit: wait you still have the issue with HDR disabled? So the dashboard still pops up but is blank, it isn't crashing? Does your machine have any special custom graphics or gaming software?

**Comment by Pololot64** [comment]
@wmsphd Do you mind right clicking the icon, resetting the settings to default and changing the acrylic enabled flag to false again? It seems like your settings could be corrupted.

**Comment by wmsphd** [comment]
Hi –

 

Thanks. I did that several times. I reset the settings and started the app and got a blank dashboard. Then I went into the config and turned off acrylic, started the app, and got nothing but an icon in the taskbar – no dashboard at all.

 

I also uninstalled and reinstalled several times.

 

I’d like to do a completely clean installation, but I’m not sure how to remove everything.. 

 

Is there a non-store version? Apps from the store keep way too much inaccessible, making troubleshooting a problem.

 

I appreciate the quick reply – and I really like the app on my other PC. 

 

 

Thanks!!

**Comment by Pololot64** [comment]
Hmm, Can you please make sure that the settings file has no personal info and then share it here? @wmsphd

**Comment by rexmtorres** [comment]
Hi.  I encountered the same issue and disabling acrylic fixed the dashboard issue.  However, when I launch shortcut maker, it exhibits the same problem.  It looks empty, but when you try to take a screenshot, it will show its contents.

![Image](https://github.com/user-attachments/assets/42964543-e6f1-4b46-9575-9e120c906ff8)

<img width="608" height="484" alt="Image" src="https://github.com/user-attachments/assets/773deab5-411e-4ea0-b070-22e3f96872ff" />

**Comment by Pololot64** [comment]
@rexmtorres  Try setting this to off: 

<img width="1137" height="735" alt="Image" src="https://github.com/user-attachments/assets/a00b4e7a-6861-4bc4-9f71-b2450036873f" />
